.circleci/config.yml

```diff
@@ -270,6 +271,16 @@ jobs:
            working_directory: /tmp/site
            command: tar -xzf docs.tar.gz && sftp -oStrictHostKeyChecking=no -b /root/sftp.txt 3472291@sftp.dc2.gpaas.net
 
+   aws-sync:
+     docker:
+       - image: amazon/aws-cli
+     steps:
+       - attach_workspace:
+           at: /tmp/site
+       - run:
+           name: Upload docs
+           working_directory: /tmp/site
+           command: tar -xzf deep-docs.tar.gz && aws s3 sync docs s3://please-build && aws s3 sync linux_amd64 s3://please-build/linux_amd64 && aws s3 sync darwin_amd64 s3://please-build/darwin_amd64 && aws s3 sync freebsd_amd64 s3://please-build/freebsd_amd64
```

Comment:
You don't have all the GPG signatures etc here, those are generated in the release step

Reply:
Ah right I see... I guess the current server just routes requests so it doesn't need to worry about that. Probably needs some thought. I guess I'll come back to the releases. We can get the docs updating first.